app/controllers/settings/users/user.js

```diff
@@ -0,0 +1,10 @@
+import Controller from '@ember/controller';
+
+export default Controller.extend({
+
+  actions: {
+    saveChanges() {
+      console.log('changing logs');
```

Comment:
Wait. What is this? The entire save functionality does... nothing?

Reply:
Nice catch. my bad